src/components/stepforms/ToDateStepForm.vue

```diff
@@ -63,5 +118,15 @@ export default class ToDateStepForm extends BaseStepForm<ToDateStep> {
     }
     this.editedStep.column = colname;
   }
+
+  updateStepFormat(newFormat: FormatOption) {
+    if (newFormat.format === 'guess') {
+      this.editedStep.format = undefined;
+    } else if (newFormat.format === 'custom') {
+      this.editedStep.format = '';
+    } else {
+      this.editedStep.format = newFormat.format;
+    }
+  }
```

Comment:
Same comment than for the dateToString step: I suggest to make the `editedStep` a computed of two data edited by `Autocomplete` and `InputText` and the component archi will be much more clearer.

Reply:
Same problem as in `FromDateStepForm`... :(